### PR TITLE
Add modal to card

### PR DESCRIPTION
**Pull request by psean-eco**
changes card component to a class component and adds a modal to it.

changes were made in Card.js

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -1,28 +1,56 @@
...
-        <p>
-          {movie.vote_average}
-          <svg height="20" width="23" class="star rating" data-rating="1">
-            <polygon points="9.9, 1.1, 3.3, 21.78, 19.8, 8.58, 0, 8.58, 16.5, 21.78" />
-          </svg>
-        </p>
-      </span>
-    </div>
-  );
+    this.handleOpenMovieModal = this.handleOpenMovieModal.bind(this);
```
Comment (naguigui) [inline src/components/Card.js:12]: Don't need to bind this if our methods are ES6 arrow functions!

**Comment by naguigui** [comment]
LGTM! Just left one minor nit lol

File: src/components/Card.js [hunk 2]
```diff
@@ -1,28 +1,56 @@
...
+    this.handleCloseMovieModal = this.handleCloseMovieModal.bind(this);
+  }
+
+  handleOpenMovieModal() {
+    this.setState({ showMovieDetailsModal: true });
+  }
```
Comment (jenny-zhong) [inline src/components/Card.js:18]:
```suggestion
  handleOpenMovieModal = () => {
    this.setState({ showMovieDetailsModal: true });
  }
```

File: src/components/Card.js (hunk 1 above)
Comment (psean-eco) [reply src/components/Card.js:12]: updated

**Pull request merged**